tools/ci/tc/decision.py

```diff
@@ -101,14 +100,13 @@ def get_extra_jobs(event):
     return jobs
 
 
-def filter_jobs(jobs, event):
-
+def filter_tasks(tasks, event):
```

Comment:
Maybe this should be `filter_pr_submitter` or something.

Reply:
I changed it to match the yaml implementation, `filter_excluded_users`. Hopefully this is a better name based on the more generic implementation.